frontend/src/pages/modelServing/screens/projects/ModelServingPlatform.tsx

```diff
@@ -119,13 +156,13 @@ const ModelServingPlatform: React.FC = () => {
         }
         description={
           shouldShowPlatformSelection && emptyModelServer
-            ? 'Select the type of model serving platform to be used when deploying models in this project.'
+            ? 'Select the model serving type to be used when deploying models from this project.'
             : undefined
         }
         popover={
           <Popover
             headerContent="About model serving"
-            bodyContent="Deploy a trained data science model to serve intelligent applications with an endpoint that allows apps to send requests to the model."
+            bodyContent="Deploy models to test them and integrate them into applications. Deploying a model makes it accessible via an API, enabling you to return predictions based on data inputs."
```

Comment:
The body content here is correct, but the `headerContent` should also be updated to "About models"